common/v2/features/Settings/components/GeneralSettings.tsx

```diff
@@ -94,31 +102,48 @@ export default class GeneralSettings extends React.Component<SettingsProps> {
     }
   };
 
+  public renderTooltipContent = () => <div>Test</div>;
+
   public render() {
     const { globalSettings } = this.props;
     return (
-      <DashboardPanel heading="General Settings">
+      <DashboardPanel heading={translateRaw('SETTINGS_GENERAL_LABEL')}>
         <Divider />
         <SettingsField>
-          <SettingsLabel>Account Settings</SettingsLabel>
+          <SettingsLabel>
+            {translateRaw('SETTINGS_HANDLING_LABEL')}
+            <Tooltip tooltip={<span>{translateRaw('SETTINGS_TOOLTIP')}</span>}>
+              {props => (
```

Comment:
This wrapper FC seems unnecessary. If so, please remove

Reply:
I couldn't get it to work without it for some reason and saw the tooltip implemented by someone else in the same way elsewhere in v2. I tried to play around a little with it but didn't want to spend too much time trying to figure out why it behaves like that if it worked :P

Reply:
ok, let's see if updating the package solves the issue.
The styling of the tooltip (size of font and container, + positioning of the icon which needs to be aligned with the copy) also needs to be fixed.